youtube-client-app/src/app/core/components/sort-panel/sort-panel.component.ts

```diff
@@ -0,0 +1,50 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-sort-panel',
+  templateUrl: './sort-panel.component.html',
+  styleUrls: ['./sort-panel.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class SortPanelComponent {
+  private date: string = 'up';
```

Comment:
please add enum sorting { up = 'up', down = 'down'} and use sorting.up instead of 'up'